### PR TITLE
Graph

### PR DESCRIPTION
**Pull request by lhchandler4**
- [ X] Top-level README “Table of Contents” is updated
 - [ X] Feature tasks for this challenge are completed
 - [ X] Unit tests written and passing
     - [ X] “Happy Path” - Expected outcome
     - [ X] Expected failure
     - [ X] Edge Case (if applicable/obvious)
 - [ X] README for this challenge is complete
     - [ X] Summary, Description, Approach & Efficiency, Solution
     - [ X] Link to code
     - [ X] Picture of whiteboard

### CODE REVIEW

File: challenges/src/main/java/Java/Graph/Edge.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package Java.Graph;
+
+public class Edge<T> {
+
```
Comment (evanslaton) [inline challenges/src/main/java/Java/Graph/Edge.java:4]: Extra lines!

File: challenges/src/main/java/Java/Graph/Graph.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+    public HashSet<Edge> getNeighbors(Node<T> node){
+        return node.neighbors;
+    }
+
+    public int getNodeNum(){
+        return this.nodes.size();
+    }
+
+//    public int size(Node startingNode){
```
Comment (evanslaton) [inline challenges/src/main/java/Java/Graph/Graph.java:42]: Corpse code!!!
